lib/core/src/lib/comments/comment-list/comment-list.component.spec.ts

```diff
@@ -29,6 +29,15 @@ describe('CommentListComponent', () => {
     let fixture: ComponentFixture<CommentListComponent>;
     let testingUtils: UnitTestingUtils;
 
+    /**
+     * Helper to retrieve the avatar image element from the DOM.
+     *
+     * @returns The avatar image element.
+     */
+    function getAvatarImg() {
+        return testingUtils.getByCSS('.adf-people-img');
```

Comment:
I see there is one more usage of that string in 162nd line. To avoid that you should put that string .adf-people-img into some const and then use that const here and also in 162nd line.